src/feeds.js

```diff
@@ -366,6 +370,60 @@ feeds.fetchEventsFromCalendar_ = function(feed, callback) {
 };
 
 
+/**
+ * Updates the notification alarms
+ */
+feeds.updateNotification = function() {
+  if (!options.get(options.Options.NOTIFICATION_SHOW)) {
+    return;
+  }
+  // If event deleted, then delete alarm
+  if(feeds.nextEvents.length === 0) {
+    chrome.alarms.getAll(function(alarms) {
+      if(alarms.length > 0) {
+        chrome.alarms.clearAll();        
+      }
+    });
+    return;
+  }
+
+  chrome.alarms.getAll(function(alarms) {
+    // Check against one event at the time
+    for(var i = 0; i < feeds.nextEvents.length; i++) {
+      if(feeds.nextEvents[i].reminders.length === 0) return;
+      // Check if the event has an alarm. This also saves the index of the alarm
+      var alarmIndex = 0;
+      var hasEventAnAlarm = alarms.some(function(alarm, index) {
+        alarmIndex = index;
+        return feeds.nextEvents[i].event_id === alarm.name;
+      });
+
+      var MINUTES_BEFORE_ALARM = feeds.nextEvents[i].reminders[0].minutes * 60 * 1000;
```

Comment:
TIME_UNTIL_ALARM_MS (the unit is milliseconds, not minutes, so the current name is a misnomer).